README.md

```diff
@@ -209,4 +209,50 @@ Logging has been set up to store logs at info level and above. Logs are stored a
 - Error logs set up to record if tasks do not behave as expected e.g. does not change status to started when the corresponding button is clicked.
 -Warning logs set to record the user ID when an authorised user logs into the app, when a new user visits the app or when a user with no write access tries to edit the tasks
 
+## Minikube
+The todo app runs locally using minikube. To run the app via Minikube run the following commands:
+
+- minikube start
+This will start the minikube cluster
+
+- kubectl port-forward service/todo-app 5000:8000
+This will run the app, to view it navigate to localhost:5000 
+
+As this is running locally, its using the secrets outlined in the .env file.
+To see what secrets have been set run:
+- kubectl get secrets
+Should display something like this:
+NAME               TYPE     DATA   AGE
+github-client      Opaque   1      7d22h
+loggly             Opaque   1      7d22h
+mongo-connection   Opaque   1      7d22h
+secret-key         Opaque   1      7d22h
+todo-secrets       Opaque   1      7d22h
+
+To edit an existing secret run:
+- kubectl edit secrets <//name of secret>
+This will bring up text file, get the value of the secret which will be in Base64, you'll need to decode it, change it and then code it back again before saving it. 
+
+To add a new secret run:
+kubectl create secret generic todo-secrets(change this) --from-literal=GITHUB_CLIENT_SECRET(change this)='the-actual-secret'
```

Comment:
Personal preference rather than requirement, but these commands can be fiddly - offering a template file for secrets can be nicer (e.g. the below) that we check in as `secrets.yaml.template` or something, as long as we're confident we/others won't check in secrets!
```
apiVersion: v1
kind: Secret
metadata:
  name: github-client
type: Opaque
data:
  backend-GITHUB_CLIENT_ID: <add_base64_secret>
---
apiVersion: v1
kind: Secret
metadata:
  name: todo-secrets
type: Opaque
data:
  backend-GITHUB_CLIENT_SECRET: <add_base64_secret>
---
apiVersion: v1
kind: Secret
metadata:
  name: mongo-connection
type: Opaque
data:
  backend-MONGO_CONNECTION_STRING: <add_base64_secret>
---
apiVersion: v1
kind: Secret
metadata:
  name: loggly
type: Opaque
data:
  backend-LOGGLY_TOKEN: <add_base64_secret>
---
apiVersion: v1
kind: Secret
metadata:
  name: secret-key
type: Opaque
data:
  backend-SECRET_KEY: <add_base64_secret>
```